reconcile/utils/amtool.py

```diff
@@ -0,0 +1,43 @@
+import tempfile
+from subprocess import run, PIPE
+
+
+class AmtoolResult(object):
+    '''This class represents a amtool command execution result'''
+    def __init__(self, is_ok, message):
+        self.is_ok = is_ok
+        self.message = message
+
+    def __str__(self):
+        return str(self.message).replace('\n', '')
+
+    def __bool__(self):
+        return self.is_ok
+
+
+def check_config(yaml_str):
```

Comment:
would it make sense to wrap this up into a `Amtool` class? that will help you get rid of the helper function in the integration.

Reply:
That would be a way to do it, yes. But I don't see the `check_config` helper function out of the integration code as it deals with the data structures that are in the integration and should never be part of the `Amtool` class. So at the moment, I don't see much benefit in changing the shape of this file